setup.py

```diff
@@ -44,6 +44,7 @@
         "papermill>=1.2.0",
         "ipykernel>=5.0.0",
         "black==19.10b0",
+        "retry==0.9.2",
```

Comment:
?

Reply:
For this https://github.com/lyft/flytekit/pull/173/files#diff-fcefe0b613289d8720e4090b8f4caa85R15-R16

Reply:
@kumare3 ^

Reply:
SageMaker suggests that hostname information might not be immediately available to the processing container. They recommend adding a retry policy on hostname resolution operations as nodes become available in the cluster.
See https://docs.aws.amazon.com/sagemaker/latest/dg/build-your-own-processing-container.html#byoc-config

I added a comment to document this